src/compiler/parse/state/tag.ts

```diff
@@ -194,6 +195,26 @@ export default function tag(parser: Parser) {
 
 		element.expression = definition.value[0].expression;
 	}
+	
+	if (name === 'svelte:element') {
+		const index = element.attributes.findIndex(attr => attr.type === 'Attribute' && attr.name === 'tag');
+		if (!~index) {
```

Comment:
using the `~` here is a little clever for my taste

Reply:
Agreed. Seems this was copy pasted straight from the `<svelte:component>` case a few lines up tho. Should we change both, neither, or just this case?

Reply:
Ah. I would say we might as well change both

Reply:
Not sure why this was even here in the first place. I doubt it is even faster considering it is doing more work.

Reply:
> There are unaddressed comments and merge conflicts on this PR. Those would need to be addressed before this could be merged

Ignore my question about styles/classes not being picked up by the dynamic elements, it's not critical.

I'm currently using this fork on a page-builder project i'm working on. The feature works flawlessly, and i'm sure the class issue could be resolved at a later stage. I'd rather this get pulled sooner rather than later